src/main/java/ToDos.java

```diff
@@ -0,0 +1,11 @@
+public class ToDos extends Task {
```

Comment:
Perhaps use singular form (ToDo) since plural form usually represents a collection of objects, eg. List